packages/core-components/api-report.md

```diff
@@ -165,6 +190,14 @@ export interface CodeSnippetProps {
   text: string;
 }
 
+// @public
+export const ConfigBasedAutoLogoutProvider: ({
```

Comment:
This no longer exist in the code. I believe you need to regenerate the API reports.